content/id/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -0,0 +1,414 @@
+---
+title: Konfigurasi Quality of Service untuk Pod
+content_type: task
+weight: 30
+---
+
+<!--
+title: Configure Quality of Service for Pods
+content_type: task
+weight: 30
+-->
+
+<!-- overview -->
+
+<!--
+This page shows how to configure Pods so that they will be assigned particular
+Quality of Service (QoS) classes. Kubernetes uses QoS classes to make decisions about
+scheduling and evicting Pods.
+-->
+Laman ini menunjukkan bagaimana mengonfigurasi Pod agar ditempatkan pada kelas _Quality of Service_ (QoS) tertentu.
+Kubernetes menggunakan kelas QoS untuk membuat keputusan tentang
+penjadwalan dan pengeluaran Pod.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+<!--
+## QoS classes
+
+When Kubernetes creates a Pod it assigns one of these QoS classes to the Pod:
+-->
+## Kelas QoS  {#qos-classes}
+
+Saat Kubernetes membuat Pod, ia menempatkan salah satu kelas QoS berikut ke Pod:
```

Comment:
"ia" seems off in bahasa indonesia since it usually refers to person, how about rephrase a bit:
```suggestion
Saat membuat Pod, Kubernetes menempatkan salah satu kelas QoS berikut untuknya:
```